OpenStack-Rabbit-Consumer/rabbit_consumer/openstack_api.py

```diff
@@ -71,7 +71,7 @@ def get_server_networks(vm_data: VmData) -> List[OpenstackAddress]:
     return OpenstackAddress.get_internal_networks(server.addresses)
 
 
-def get_metadata(vm_data: VmData) -> dict:
+def get_server_metadata(vm_data: VmData) -> dict:
```

Comment:
Is this method used anywhere else for the consumers and has the change in method name been updated elsewhere (in the consumers and the unit tests) where the method is called?

From what I can see this method isn't being currently used?

Reply:
I've just double checked, it's called in `message_consumer`